grpc-proto/proto/query.proto

```diff
@@ -86,14 +74,13 @@ message Value {
     double double = 5; // double
     bool boolean = 6; // boolean
     string string = 7; // ascii, varchar, text
-    bytes bytes = 8; // blob, custom
-    Inet inet = 9; // inet
-    Uuid uuid = 10; // uuid, timeuuid
-    Date date = 11; // date
-    Time time = 12; // time
-    Collection collection = 13; // list, set, tuple
-    Map map = 14;
-    UdtValue udt = 15;
+    bytes bytes = 8; // blob, inet, custom
+    Uuid uuid = 9; // uuid, timeuuid
+    uint32 date = 10; // date
+    uint64 time = 11; // time
```

Comment:
Is it trivial for clients to encode date and time values? Could you add a comment about the expected representation here?